libs/sdk-ui-kit/src/Header/generateHeaderMenuItemsGroups.ts

```diff
@@ -15,6 +15,7 @@ export function generateHeaderMenuItemsGroups(
     tabId?: string,
     hasNoDataSet?: boolean,
     backendSupportsDataItem?: boolean,
+    disableKPIDashboards?: boolean,
```

Comment:
We should call this `disableAnalyticalDashboards` since on of the other parameters is called `hasAnalyticalDashboards`.